django_prices_openexchangerates/templatetags/prices_multicurrency.py

```diff
@@ -8,20 +9,23 @@
 
 @register.simple_tag
 def gross_in_currency(price, currency, **kwargs):
```

Comment:
This no longer makes sense. You can do `{{ price.gross|in_currency:'USD' }}`. What's the point of converting both net and gross and then immediately discarding one of them?

Reply:
This PR is still in progress. For now tests are updated to pass.